Implemented: Added new inventory cycle count feature for warehouse.(OFBIZ-10577)

yashwantdhakad opened the pull request:
Implemented: Added new inventory cycle count feature for warehouse.
(OFBIZ-10577)
Added new inventory cycle count and user can create new cycle count session from pending location, find the existing cycle count session and review the cycle count session and see progress report of the cycle count. Along with we have locked the facility location until cycle count is not completed and also updated inbound and outbound transaction services of inventory item if facility location is locked in any of active cycle count session.
Thanks: Ankit Joshi for your help

Commit:
Implemented: Added new inventory cycle count feature for warehouse.
(OFBIZ-10577)
Added new inventory cycle count and user can create new cycle count session from pending location, find the existing cycle count session and review the cycle count session and see progress report of the cycle count. Along with we have locked the facility location until cycle count is not completed and also updated inbound and outbound transaction services of inventory item if facility location is locked in any of active cycle count session.
Thanks: Ankit Joshi for help

yashwantdhakad committed:
Merge pull request #2 from apache/trunk

Updated the fork branch with main apache branch

PierreSmits commented:
I tried to merge the PR with a clean branch (based on latest in trunk) yesterday, and got a merge conflict on applications/product/config/ProductUiLabels.xml.

Please have a look at this.

Commit:
Fixed: Resolved the conflicts due to code merge.

yashwantdhakad committed:
Merge pull request #4 from apache/trunk

Merge latest changes from OFBiz trunk.

pwverma commented on applications/product/template/cyclecount/CountReport.ftl:
```diff
@@ -0,0 +1,242 @@
+<#--
```
This file needs formatting :)

pwverma commented on applications/product/template/cyclecount/CountReport.ftl:
```diff
@@ -0,0 +1,242 @@
+                        <div class="screenlet">
+                            <div class="screenlet-title-bar">
+                                <ul>
+                                    <li class="h3">${uiLabelMap.CommonSearchOptions}/li>
```
We missed closing the li tag properly.

pwverma commented on applications/product/template/cyclecount/PendingLocations.ftl:
```diff
@@ -0,0 +1,115 @@
+              <th>${uiLabelMap.ProductSection}</th>
+              <th>${uiLabelMap.ProductLevel}</th>
+              <th>${uiLabelMap.ProductPosition}</th>
+              <th>Last counted on</th>
```
Need to convert it with uiLabel.

yashwantdhakad replied:
Done.

pwverma commented on applications/product/template/cyclecount/ReviewCountSession.ftl:
```diff
@@ -0,0 +1,188 @@
+        }
+    }
+</script>
+<#if countSessionValue?has_content>
```
Needs formatting here also

pwverma commented:
Also, please replace module with MODULE as per the consistency :)

Commit:
Merge branch 'trunk' into OFBIZ-10577

yashwantdhakad committed:
Merge pull request #5 from apache/trunk

Merge the apache ofbiz trunk into my fork repository

Commit:
Fixed: The conflicts during merge from trunk to feature branch.

Commit:
Fixed: Fixed the following issues: 1. Added missing UI labels. 2. Fixed the checkbox selection issue. 3. Fixed the inventory item lookup dialog in record count screen. 4. Done some formatting changes as per the best practice.

yashwantdhakad committed:
Update OrderList.ftl

Revert the changes added by mistakenly.

yashwantdhakad committed:
Merge pull request #6 from apache/trunk

Merge the trunk branch into fork branch

Commit:
Merge branch 'trunk' into OFBIZ-10577

Commit:
Merge branch 'OFBIZ-10577' of https://github.com/yashwantdhakad/ofbiz-framework into OFBIZ-10577

yashwantdhakad committed:
Merge pull request #7 from apache/trunk

Merged the Apache branch with my fork repository

Commit:
Fixed: Resolved the conflicts
(OFBIZ-10577)
Resolved the conflicts due to merge from OFBiz trunk branch to fork repository.

yashwantdhakad committed:
Merge pull request #8 from apache/trunk

Merged apache trunk code to fork repository.

yashwantdhakad committed:
Merge pull request #9 from yashwantdhakad/trunk

Improved: Merged the latest code.

yashwantdhakad committed:
Merge pull request #10 from apache/trunk

Updated the branch with latest code.

Commit:
Merge branch 'trunk' into OFBIZ-10577

Commit:
Merge branch 'OFBIZ-10577' of https://github.com/yashwantdhakad/ofbiz-framework into OFBIZ-10577

yashwantdhakad committed:
Merge pull request #11 from yashwantdhakad/trunk

Updated the branch with latest code.

Commit:
Merge branch 'OFBIZ-10577' of https://github.com/yashwantdhakad/ofbiz-framework into OFBIZ-10577

Commit:
Fixed: Formatting changes
(OFBIZ-10577)
Done some formatting changes to remove some of style errors in java files.

yashwantdhakad committed:
Merge branch 'apache:trunk' into OFBIZ-10577

JacquesLeRoux commented:
Hi, what is the status here?
